app/models/announcement/templates/monthly.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+class Announcement
+  module Templates
+    class Monthly
+      include ApplicationHelper
+
+      def initialize(event:, author:)
+        @event = event
+        @author = author
+      end
+
+      def title
+        "#{Date.current.strftime("%B, %Y")} Update"
+      end
+
+      def json_content
+        {
+          type: "doc",
+          content: [
+            { type: "paragraph", content: [{ type: "text", text: "Hey all!" }] },
+            {
+              type: "paragraph",
+              content: [
+                {
+                  type: "text",
+                  text: "Thank you for your support and generosity! With this funding, we'll be able to better work towards our mission.",
+                },
+              ],
+            },
+            {
+              type: "paragraph",
+              content: [
+                {
+                  type: "text",
+                  text: "We'd like to thank all of the donors from the past month that contributed towards our organization:",
+                },
+              ],
+            },
+            { type: "donationSummary", attrs: { startDate: Date.current.beginning_of_month.. } },
```

Comment:
Before the end of the month, we need to handle the case where an org has zero donatoins